operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -362,6 +362,11 @@ metadata:
     categories: Streaming & Messaging
     certified: 'false'
     containerImage: registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:95f5aa75cd1f7228e78fd4d88d786713fba4cf828dc22bc2dd1d0380909c1aef
+    operandImages: |
+      registry.redhat.io/amq-streams/bridge-rhel9@sha256:4db7b231d68a244f5f71c8587123890e1344be933e1a030097c4b32a9bbfe32d
+      registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
+      registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:050aed7a91af406fbafcf6ce8fff4fa451cf4855f74a91aa081d794bf8c9aaa3
+      registry.redhat.io/amq-streams/maven-builder-rhel9@sha256:2393070a07677a482e90f53fef58f12e1c73963cf164738aaa1c095e5fe77d0d
```

Comment:
I'm not entirely sure it will work this way. You may need to have a separate annotation that for each that can be used by the env values. For example:
```yaml
annotations:
  operator-image: |
    registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:06de6eaeb8cae4ea31f05a427bbe53f9d078de0030bd56fce4ecfbb5d0be461a
  kafka-images: |
    3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
    3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:42bf60ce31540dd61fab2c9886d791e41f063ea6f86628694b9e60e49bc8951b
  kafka-current-image: |
    registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:42bf60ce31540dd61fab2c9886d791e41f063ea6f86628694b9e60e49bc8951b
  kafka-previous-image: |
    registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
    
```

Then reference them like:
```yaml
- name: STRIMZI_KAFKA_IMAGES
  valueFrom:
    fieldRef:
      fieldPath: metadata.annotations['kafka-images']
```

Also, I think the annotations need to be on the pod template rather than the root CSV so that the ENVs can reference them.